recipe/meta.yaml

```diff
@@ -7,20 +7,19 @@ package:
 source:
   fn: dask-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
-  sha256: 5e7876bae2a01b355d1969b73aeafa23310febd8c353163910b73e93dc7e492c
+  sha256: 64d59fa0dc5119fab0ea0deb7106ef01747ae9d10eb8fc3ef9477e7bca1062c8
 
 build:
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
-  noarch: python
 
 requirements:
   build:
-    - python
+    - python >= 3.5
```

Comment:
No space here I think. And on line 22

```
- python >=3.5
```

Reply:
Oops, sorry for the extra space above. Thanks for catching that Tom. 🙂